### DISCUSSION TITLE
Could you add `groupByYear` an optional front matter parameter ?

### DISCUSSION THREAD

**Discussion by slashformotion** [jpanther/congo#21] [Ideas]
rn it's a site wide parameter available in `params.list.groupByYear`.
It would be great to be able to override the value set in the site params from the front matter. (see exemple bellow)

```
title: "Cheatsheets"
description: "Cheatsheets i find usefull."

groupByYear: true
```
![image](https://user-images.githubusercontent.com/45801817/138960671-f75848d9-f043-4b7e-848f-0e7ffe002894.png)

```
title: "Cheatsheets"
description: "Cheatsheets i find usefull."

groupByYear: false
```
![image](https://user-images.githubusercontent.com/45801817/138960742-e1e1f7ea-d2ba-40b2-a590-7190d3aaa4bd.png)

**Comment by jpanther** [comment]
Thanks for all the suggestions! This was an easy change and has now been included in the dev branch. It will be in the next release.

_Ref: 14a6aba0b556d278f09a8271202ee3d2c5d8c791_

**Reply by jpanther** [reply]
This is now available in v1.5.0
